src/lib/utilities/route-for-api.ts

```diff
@@ -116,6 +116,7 @@ export function routeForApi(
     'workflow.terminate': `/namespaces/${parameters?.namespace}/workflows/${parameters?.workflowId}/runs/${parameters?.runId}/terminate`,
     'workflow.cancel': `/namespaces/${parameters.namespace}/workflows/${parameters.workflowId}/runs/${parameters.runId}/cancel`,
     'workflow.signal': `/namespaces/${parameters.namespace}/workflows/${parameters.workflowId}/runs/${parameters.runId}/signal`,
+    'workflow.reset': `/namespaces/${parameters.namespace}/workflows/${parameters.workflowId}/runs/${parameters.runId}/reset`,
```

Comment:
Just double checking, this route is in ui-server already or is that a separate PR coming?

Reply:
Yep, it already exists